modules/util/util.go

```diff
@@ -137,36 +137,36 @@ func MergeInto(dict map[string]interface{}, values ...interface{}) (map[string]i
 	return dict, nil
 }
 
-// RandomInt returns a random integer between 0 and limit, inclusive
-func RandomInt(limit int64) (int64, error) {
+// SecureRandomInt returns a secure random integer between 0 and limit, inclusive
+func SecureRandomInt(limit int64) (int64, error) {
 	rInt, err := rand.Int(rand.Reader, big.NewInt(limit))
 	if err != nil {
 		return 0, err
 	}
 	return rInt.Int64(), nil
 }
 
-const letters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+const randomLetters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
 
-// RandomString generates a random alphanumerical string
-func RandomString(length int64) (string, error) {
-	bytes := make([]byte, length)
-	limit := int64(len(letters))
-	for i := range bytes {
-		num, err := RandomInt(limit)
+// SecureRandomString generates a secure random alphanumerical string, each byte is generated by [0,61] range
+func SecureRandomString(length int64) (string, error) {
```

Comment:
People shouldn't get the idea that this can be used for security generated things like hash and random strings, the `SecureRandomBytes` + encoding should be used for that. So I'm not sure about this, as it's only secure in the sense that's cryptographic random.

Reply:
Then what about `CryptoRandomString`? Indeed it's different from `MathRandomString` because is much slower and more secure.

Reply:
Updated, please review again